Dockerfile-dev

```diff
@@ -0,0 +1,64 @@
+FROM php:7.4-cli
+
+LABEL maintainer="Tobias Munk <tobias@diemeisterei.de>"
+
+ENV CODECEPT_USER codecept
+
+# Install required system packages
+RUN apt-get update && \
+    apt-get -y install \
+            git \
+            zlib1g-dev \
+            libmemcached-dev \
+            libpq-dev \
+            libssl-dev \
+            libxml2-dev \
+            libzip-dev \
+            unzip \
+        --no-install-recommends && \
+        apt-get clean && \
+        rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Install php extensions
+RUN docker-php-ext-install \
+    bcmath \
+    mysqli \
+    pdo pdo_mysql pdo_pgsql \
+    soap \
+    sockets \
+    zip
```

Comment:
The only database extension used by tests is sqlite.
Github Actions enable these extensions - curl, mbstring, openssl, pdo, pdo_sqlite.
Other extensions aren't necessery in Dockerfile which runs tests.